src/reanimated2/hook/useAnimatedRef.ts

```diff
@@ -19,6 +22,25 @@ function getShareableShadowNodeFromComponent(
   return getShadowNodeWrapperFromHostInstance(component);
 }
 
+function getScrollableRef(component: ComponentRef): ComponentRef {
```

Comment:
getScrollableRef -> getComponentRef

Reply:
`getComponentRef` is too generic I think